scripts/src/create.ts

```diff
@@ -3,14 +3,23 @@
  * All the names of companies, products and KIN values are completely made up and are used for TESTING only.
  */
 import * as fs from "fs";
-import { User, AuthToken } from "./models/users";
+import { AuthToken, User } from "./models/users";
 import { Application } from "./models/applications";
-import { Offer, OfferContent, AppOffer, Asset, OfferOwner } from "./models/offers";
-import { Order } from "./models/orders";
+import { AppOffer, Asset, Offer, OfferContent, OfferOwner } from "./models/offers";
+import { OpenOrder, Order } from "./models/orders";
 
 import { init as initModels } from "./models";
 import { getConfig } from "./config";
-import { animalPoll, kikPoll, tutorial, Poll, Tutorial, CouponInfo, CouponOrderContent } from "./services/offer_contents";
+import {
+	animalPoll,
+	CouponInfo,
+	CouponOrderContent,
+	kikPoll,
+	Poll,
+	tutorial,
+	Tutorial
+} from "./services/offer_contents";
```

Comment:
I like it the way it was before, a one liner, it's consistent with our other imports.

Reply:
The auto-formatter does this